cluster-autoscaler/cloudprovider/azure/test/e2e/e2e_suite_test.go

```diff
@@ -59,7 +64,8 @@ var (
 )
 
 func init() {
-	flag.StringVar(&resourceGroup, "resource-group", "", "resource group containing cluster-autoscaler-managed resources")
+	flag.StringVar(&nodeResourceGroup, "node-resource-group", "", "resource group containing cluster-autoscaler-managed resources")
+	flag.StringVar(&clusterResourceGroup, "cluster-resource-group", "", "cluster resource group contains the managed cluster we end up creating")
```

Comment:
this is unused now, but we will need it in the future and maeks it clearer which rg we are using.

Reply:
what's a use case for this in the future?

Reply:
any usecase involving using the mc  arm clients or requiring a lookup of the mc